README.md

```diff
@@ -1,2 +1,31 @@
 # aws-synthetic-checks
+
 AWS re-usable synthetic uptime &amp; graphql operation checks stored in S3
+
```

Comment:
I can add some descriptions here in a future pr, the most important being how to configure various synthetics via the environment variables used here.